package.json

```diff
@@ -58,9 +60,17 @@
     "autoprefixer": "10.4.2",
     "better-docs": "2.7.1",
     "buffer": "6.0.3",
+    "crypto-browserify": "3.12.0",
+    "https-browserify": "1.0.0",
+    "os-browserify": "0.3.0",
     "postcss": "8.4.5",
+    "process": "0.11.10",
+    "react-app-rewired": "2.1.11",
     "redux-devtools-extension": "2.13.9",
     "stream": "0.0.2",
-    "tailwindcss": "3.0.15"
+    "stream-browserify": "3.0.0",
+    "stream-http": "3.2.0",
+    "tailwindcss": "3.0.15",
+    "url": "0.11.0"
```

Comment:
Can you elaborate what packages/code needed these dependencies? 

I know we covered it briefly, just so I can get a whole picture. Were these all being used by MadNetJS?